app/services/publishing_api_payload.rb

```diff
@@ -34,6 +34,15 @@ def payload
 
 private
 
+  def links
+    associations = document.associations
+    if associations.include?("primary_publishing_organisation") && !associations.include?("original_primary_publishing_organisation")
```

Comment:
We can simplify this for now as the original org will never be there.

Reply:
well if someone updates the primary_publishing_organisation we don't want to overwrite the original? this way it'll save the original once the document is first sent off to publishing-api.

Reply:
I think you're right, but I also think we should leave that for now until we properly support a history. Also, if the user accidentally chose the wrong org initially, then it's valid to overwrite it.